lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.BoilerHotWater.rb

```diff
@@ -0,0 +1,106 @@
+class ASHRAE901PRM < Standard
+  # @!group BoilerHotWater
+
+  # find search criteria
+  #
+  # @return [Hash] used for standards_lookup_table(model)
+  def boiler_hot_water_find_search_criteria(boiler_hot_water)
+    # Define the criteria to find the boiler properties
+    # in the hvac standards data set.
+    search_criteria = {}
+    search_criteria['template'] = template
+
+    # Get fuel type
+    search_criteria['fuel_type'] = 'Gas'
+
+    # Get the fluid type
+    search_criteria['fluid_type'] = 'Hot Water'
+
+    return search_criteria
+  end
+
+  # Find capacity in W
+  #
+  # @return [Double] capacity in W
+  def boiler_hot_water_find_capacity(boiler_hot_water)
+    capacity_w = nil
+    if boiler_hot_water.nominalCapacity.is_initialized
+      capacity_w = boiler_hot_water.nominalCapacity.get
+    elsif boiler_hot_water.autosizedNominalCapacity.is_initialized
+      capacity_w = boiler_hot_water.autosizedNominalCapacity.get
+    else
+      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.BoilerHotWater', "For #{boiler_hot_water.name} capacity is not available, cannot apply efficiency standard.")
+      successfully_set_all_properties = false
+      return successfully_set_all_properties
+    end
+
+    return capacity_w
+  end
```

Comment:
For example, unless I'm missing something, this method is exactly the same as this one: https://github.com/NREL/openstudio-standards/blob/abae01541a336d08f736eb00fad9fba24fea4f23/lib/openstudio-standards/standards/Standards.BoilerHotWater.rb#L41-L54